src/Views/ListView/ListView.vala

```diff
@@ -269,15 +269,11 @@ public class Noise.ListView : ContentView, Gtk.Box {
     }
 
     public Gee.Collection<Media> get_media () {
-        var media_list = new Gee.ArrayQueue<Media> ();
-        media_list.add_all (list_view.get_table ().values);
-        return media_list;
+        return list_view.get_table ();
```

Comment:
The previous code was a soft copy of the list returned by `get_table()`. This is usually done as a defensive mechanism when possibility that the list will be mutated by the outside code, to prevent the outside code from modifying the  internal list of the `list_view`. Are you sure this will not introduce side effects and issues when the caller of `get_media` removes or adds things to the list?

Reply:
Oh, I didn't thought about that.